New evaluation.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>New Evaluation</title>
+    
+    
+<link rel="stylesheet" href="css/New%20Evaluation.css">
+</head>
+<body>
+    
+    <form id="candidate">
+        <input type="text" placeholder="Candidate" name="Candidate">
+        <input type="text" placeholder="Interviewer" name="Interviewer">
+        <input type="date">
+    </form>
+    
+    <h1>Techinical Level</h1>
+<form>
+    <table align="centre">
+        <tr>
+            <th>Trainee</th>
+            <th>Junior</th>
+            <th>Mid</th>
+            <th>Senior</th>
+        </tr>
+        <tr>
+            <td>
+                <input type="radio" name="level" value="1"></td>
+            <td>
+                <input type="radio" name="level" value="j1">
+                <input type="radio" name="level" value="j2">
+                <input type="radio" name="level" value="j3">
+            </td>
+            <td>
+                <input type="radio" name="level" value="m1">
+                <input type="radio" name="level" value="m2">
+                <input type="radio" name="level" value="m3">
+            </td>
+            <td>
+                <input type="radio" name="level" value="s1">
+                <input type="radio" name="level" value="s2">
+                <input type="radio" name="level" value="s3">
+            </td>
+        </tr>
+        
+    
+    </table>
+</form>
+    
+    <h2>Should the candidate be hired?</h2>
+    <form>
```

Comment:
I would like to know what made you consider using many forms instead of just one. I would expect that you want to do something with all the data you are gathering here. Is there any reasoning why this approach would help you?